contracts/cw20-atomic-swap/src/contract.rs

```diff
@@ -160,16 +192,37 @@ fn parse_hex_32(data: &str) -> StdResult<Vec<u8>> {
     }
 }
 
-fn send_native_tokens(from: &HumanAddr, to: &HumanAddr, amount: Vec<Coin>) -> Vec<CosmosMsg> {
+fn send_tokens<A: Api>(
```

Comment:
This is a nice one.

Let's remember this one for #73 I think there are quite a few helpers we may want to pull out.

Reply:
OK, I'll assign me #73 next.